sonic_platform_base/psu_base.py

```diff
@@ -27,7 +27,7 @@ class PsuBase(device_base.DeviceBase):
     _thermal_list = []
 
     # Status of Master LED
-    psu_master_led_color = STATUS_LED_COLOR_OFF
+    psu_master_led_color = None
```

Comment:
I think we should remove this, as it makes it a class variable, which is shared between all instances. I also think we should do the same for `_fan_list` and `_thermal_list`, moving the comments into the constructor. Thoughts?

Reply:
I agree for _fan_list and _thermal_list. But I am not so sure about psu_master_led_color, it looks like a class level variable to me.

Reply:
I don't think we want `psu_master_led_color` to be class-level; each PSU instance should have it's own LED status.

Reply:
Understood.

Reply:
I saw some comments for the class level _thermal_list. So if vendor doesn't call PsuBase.__init__, it is an issue of vendor side, right?

```python
# List of ThermalBase-derived objects representing all thermals
    # available on the PSU. Put a class level _thermal_list here to 
    # avoid an exception when call get_num_thermals, get_all_thermals
    # and get_thermal if vendor does not call PsuBase.__init__ in concrete
    # PSU class
    _thermal_list = []
```

Reply:
Hmm... Maybe we should leave this for now, then to prevent breaking anything. However, I do agree that it should be a vendor issue. We can look into doing this in a separate PR.

Reply:
It seems psu_master_led_color should be a class level variable because it is used in a classmethod. See: 

```python
    @classmethod
    def get_status_master_led(cls):
        """
        Gets the state of the Master status LED for a given device-type

        Returns:
            A string, one of the predefined STATUS_LED_COLOR_* strings.
        """
        return cls.psu_master_led_color
```

Reply:
It looks like related changes are also being made in https://github.com/Azure/sonic-platform-daemons/pull/119/files. For now, let's leave this alone and revisit it in a separate PR.

Reply:
Should I revert psu_base.py?

Reply:
No I think this is fine. We just won't worry about changing from class vars to instance vars in this PR.